### PR TITLE
Add option '--maxIssues=n' to tolerate up to 'n' unused exports.

### PR DESCRIPTION
**Pull request by ksr79**
Added option to allow a maximum number of unused exports.

https://github.com/pzavolinsky/ts-unused-exports/issues/85

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ export interface ExtraCommandLineOptions {
   exitWithCount?: boolean;
   excludeDeclarationFiles?: boolean;
   ignoreFilesRegex?: string[];
+  maxIssues: number;
```
Comment (mrseanryan) [inline src/types.ts]: I think `maxIssues` can be undefined

In which case can be with `?`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+### Added
+
+- Add options to allow maximum number of issues: --maxIssues
```
Comment (mrseanryan) [inline CHANGELOG.md]: Add options -> Add an option

**Comment by mrseanryan** [comment]
@ksr79 would it be possible to add a unit test for the new option?

To try prevent the code coverage decreasing.

Example:

https://github.com/pzavolinsky/ts-unused-exports/blob/master/features/exclude-paths-from-report.feature

File: src/types.ts (hunk 1 above)
Comment (ksr79) [reply src/types.ts]: I changed it to option, as requested.

File: CHANGELOG.md (hunk 2 above)
Comment (ksr79) [reply CHANGELOG.md]: Changed text as requested.

**Comment by ksr79** [comment]
I added two tests, hope that the coverage is fine now. Please have a look.

**Comment by mrseanryan** [comment]
@ksr79  looks good!

@pzavolinsky  I think something is wrong with the travis integration.

It looks like someone needs to fix in travis itself (I don't have permissions)

https://travis-ci.org/pzavolinsky/ts-unused-exports

**Comment by mrseanryan** [comment]
@pzavolinsky  not sure what is going on with travis.

I could open a new PR with the same commits ?

**Comment by pzavolinsky** [comment]
Sorry about this, this looks like a Travis issue: https://github.com/travis-ci/travis-ci/issues/10187

To be honest Travis is kind of a mystery sometimes. Maybe we can try creating a new PR?

**Comment by mrseanryan** [comment]
@pzavolinsky  I re-created the commits in a new PR - got green build, and merged.

@ksr79 thank you for contributing!

(closing)

**Pull request closed without merging**